README.md

```diff
@@ -114,6 +124,20 @@ The **web app** runs on any device and is mobile optimised. It provides:
 
 Adding push buttons to control volume, skipping tracks, pause, play: read the [GPIO buttons installation guide](docs/GPIO-BUTTONS.md).
 
+### Components
```

Comment:
Not sure we should have this in the readme file. It feels to detailed. Possibly just one line and then actually linking to the folder `components` in the code. Note: link would need to point to master not `develop` once we released 2.0

Reply:
Ok for me.
Links are only relative, so in master they would point to the correct master file.